tests/PHPStan/KeywordSelfRuleTest.php

```diff
@@ -24,17 +24,17 @@ public function provideRule()
             'interface' => [
                 'filePaths' => [__DIR__ . '/KeywordSelfRuleTest/TestInterface.php'],
                 'errorMessage' => "Can't use keyword 'self'. Use 'TestInterface' instead.",
-                'errorLines' => [13, 18, 18],
+                'errorLines' => [13, 13, 18, 18],
```

Comment:
Add a comment here and for the others you've updated so if we look at this in the future and actually want to fix it, it's immediately clear what's incorrectly double and what's correct.
```suggestion
                // Note that 13 only actually has one violation, but does no harm to be counted twice.
                'errorLines' => [13, 13, 18, 18],
```

Reply:
Updated